static/script/devices/mediaplayer/live/restartable.js

```diff
@@ -53,6 +54,7 @@ require.def(
         var RestartableLivePlayer = Class.extend({
             init: function() {
                 this._mediaPlayer = RuntimeContext.getDevice().getMediaPlayer();
+                this._timeUntilStartOfWindow = null;
```

Comment:
Can this be renamed `millisecondsUntilStartOfWindow` or similar please? Time in TAL media player is generally done in seconds, which could lead to confusion.